mirrord.rb

```diff
@@ -9,31 +9,24 @@ class Mirrord < Formula
   on_macos do
     url "https://github.com/metalbear-co/mirrord/releases/download/3.68.0/mirrord_mac_universal.zip"
     sha256 "2ce5d18705880cc29568b40413b10fb38adbad6a34ed0db660f7784036da56a3"
-
-    def install
-      bin.install "mirrord"
-    end
   end
 
   on_linux do
     if Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
       url "https://github.com/metalbear-co/mirrord/releases/download/3.68.0/mirrord_linux_aarch64.zip"
       sha256 "0959c1ec621558775fb8599bc591b1c19c564835190c639c19c0aa141e1b01ed"
-
-      def install
-        bin.install "mirrord"
-      end
     end
     if Hardware::CPU.intel? && Hardware::CPU.is_64_bit?
       url "https://github.com/metalbear-co/mirrord/releases/download/3.68.0/mirrord_linux_x86_64.zip"
       sha256 "7635e85e278bae890f1ab3a6e181de34cc9257d4145cc5d2dec2fd119c612ad3"
-
-      def install
-        bin.install "mirrord"
-      end
     end
   end
 
+  def install
+    bin.install "mirrord"
+    generate_completions_from_executable(bin/"mirrord", "completions")
```

Comment:
Can a maintainer pull this branch and try to reinstall mirrord using brew? I keep having brew crash and I can't tell if it's an issue with my machine or with the binary:

```bash
$ brew reinstall mirrord --verbose 
rm /opt/homebrew/bin/mirrord
==> Fetching metalbear-co/mirrord/mirrord
==> Downloading https://github.com/metalbear-co/mirrord/releases/download/3.68.0/mirrord_mac_universal.zip
Already downloaded: /Users/brad/Library/Caches/Homebrew/downloads/ddd75a75bac67bf803dbeb0e96d3b3c23655ff44cca6aab204932422422c2dbb--mirrord_mac_universal.zip
==> Verifying checksum for 'ddd75a75bac67bf803dbeb0e96d3b3c23655ff44cca6aab204932422422c2dbb--mirrord_mac_universal.zip'
==> Reinstalling metalbear-co/mirrord/mirrord 
/usr/bin/env PATH=/opt/homebrew/Library/Homebrew/shims/mac/super:/usr/bin:/bin:/usr/sbin:/sbin unzip -o /Users/brad/Library/Caches/Homebrew/downloads/ddd75a75bac67bf803dbeb0e96d3b3c23655ff44cca6aab204932422422c2dbb--mirrord_mac_universal.zip -d /private/tmp/d20230919-9439-1llqqbu
cp -p /private/tmp/d20230919-9439-1llqqbu/mirrord /private/tmp/mirrord-20230919-9439-e3ox39/mirrord
brew: exec failed: {"SHELL"=>"bash"}
ln -s ../Cellar/mirrord/3.68.0/bin/mirrord mirrord
Error: Failure while executing; `\{\"SHELL\"=\>\"bash\"\} /opt/homebrew/Cellar/mirrord/3.68.0/bin/mirrord completions bash` exited with 1. Here's the output:
```

Reply:
After switching to caveats:

```bash
$ brew reinstall mirrord 
==> Fetching metalbear-co/mirrord/mirrord
==> Downloading https://github.com/metalbear-co/mirrord/releases/download/3.68.0/mirrord_mac_universal.zip
Already downloaded: /Users/brad/Library/Caches/Homebrew/downloads/ddd75a75bac67bf803dbeb0e96d3b3c23655ff44cca6aab204932422422c2dbb--mirrord_mac_universal.zip
==> Reinstalling metalbear-co/mirrord/mirrord 
==> Caveats
mirrord has been installed but you'll need to manually generate and source
the completions for your shell like:

  source <(mirrord completions bash)

And equivalent for other shells. Check the documentation for more info:

  mirrord completions --help
==> Summary
🍺  /opt/homebrew/Cellar/mirrord/3.68.0: 3 files, 129MB, built in 2 seconds
==> Running `brew cleanup mirrord`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
```